Gordon360/Controllers/RecIM/ParticipantsController.cs

```diff
@@ -71,11 +71,21 @@ public async Task<ActionResult<ParticipantExtendedViewModel>> AddParticipant(str
         }
 
         [HttpPatch]
-        [Route("{username}")]
+        [Route("{username}/admin")]
         [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.RECIM_PARTICIPANT)]
-        public async Task<ActionResult<ParticipantExtendedViewModel>> UpdateParticipant(string username, [FromBody] bool isAdmin)
+        public async Task<ActionResult<ParticipantExtendedViewModel>> SetParticipantAdminStatus(string username, [FromBody] bool isAdmin)
+        {
+            var participant = await _participantService.SetParticipantAdminStatusAsync(username,isAdmin);
+            return CreatedAtAction(nameof(GetParticipantByUsername), new { username = participant.Username }, participant);
+        }
+
+        [HttpPatch]
+        [Route("{username}/emails")]
+        public async Task<ActionResult<ParticipantExtendedViewModel>> UpdateParticipant(string username, [FromBody] bool allowEmails)
```

Comment:
```suggestion
        public async Task<ActionResult<ParticipantExtendedViewModel>> UpdateParticipantAllowEmails(string username, [FromBody] bool allowEmails)
```
Can you update the method names here and in the service? It is no longer an update participant route or function, so the name should be updated to what it actually does.

Reply:
Originally I had an extra viewmodel to change the `SpecifiedGender` (which could be possible if the user is not from Gordon). However, I shelved since we need to talk to our customer about how we want to handle that.

Reply:
non-blocking